include/pinocchio/bindings/python/spatial/inertia.hpp

```diff
@@ -295,6 +312,251 @@ namespace pinocchio
+#ifndef PINOCCHIO_PYTHON_NO_SERIALIZATION
+          .def_pickle(Pickle())
+#endif
+          ;
+        PINOCCHIO_COMPILER_DIAGNOSTIC_POP
+      }
+
+      static Scalar getMass(const PseudoInertia & self)
+      {
+        return self.mass;
+      }
+      static void setMass(PseudoInertia & self, Scalar mass)
+      {
+        self.mass = mass;
+      }
+
+      static Vector3 getH(const PseudoInertia & self)
+      {
+        return self.h;
+      }
+      static void setH(PseudoInertia & self, const Vector3 & h)
+      {
+        self.h = h;
+      }
+
+      static Matrix3 getSigma(const PseudoInertia & self)
+      {
+        return self.sigma;
+      }
+      static void setSigma(PseudoInertia & self, const Matrix3 & sigma)
+      {
+        self.sigma = sigma;
+      }
+
+      static VectorXs toDynamicParameters_proxy(const PseudoInertia & self)
+      {
+        return self.toDynamicParameters();
+      }
+
+      static void expose()
+      {
+#if PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION == 6 && EIGENPY_VERSION_AT_LEAST(2, 9, 0)
+        typedef PINOCCHIO_SHARED_PTR_HOLDER_TYPE(PseudoInertia) HolderType;
+#else
+        typedef ::boost::python::detail::not_specified HolderType;
+#endif
+        bp::class_<PseudoInertia, HolderType>(
+          "PseudoInertia",
+          "This class represents a pseudo inertia matrix and it is defined by its mass, vector "
+          "part, and 3x3 matrix part.\n\n"
+          "Supported operations ...",
+          bp::no_init)
+          .def(PseudoInertiaPythonVisitor<PseudoInertia>())
+          .def(CopyableVisitor<PseudoInertia>())
+          .def(PrintableVisitor<PseudoInertia>());
+      }
+
+    private:
+      static Matrix4 __array__(const PseudoInertia & self, bp::object)
+      {
+        return self.toMatrix();
+      }
+
+      struct Pickle : bp::pickle_suite
+      {
+        static boost::python::tuple getinitargs(const PseudoInertia & pi)
+        {
+          return bp::make_tuple(pi.mass, pi.h, pi.sigma);
+        }
+
+        static bool getstate_manages_dict()
+        {
+          return true;
+        }
+      };
+
+    }; // struct PseudoInertiaPythonVisitor
+
+    template<typename LogCholeskyParameters>
+    struct LogCholeskyParametersPythonVisitor
+    : public boost::python::def_visitor<LogCholeskyParametersPythonVisitor<LogCholeskyParameters>>
+    {
+      enum
+      {
+        Options = LogCholeskyParameters::Options
+      };
+      typedef typename LogCholeskyParameters::Scalar Scalar;
+      typedef typename LogCholeskyParameters::Vector10 Vector10;
+      typedef typename LogCholeskyParameters::Matrix10 Matrix10;
+      typedef Eigen::Matrix<Scalar, Eigen::Dynamic, 1, Options> VectorXs;
+      typedef Eigen::Matrix<Scalar, Eigen::Dynamic, Eigen::Dynamic, Options> MatrixXs;
+
+    public:
+      template<class PyClass>
+      void visit(PyClass & cl) const
+      {
+        cl.def(bp::init<const VectorXs &>((bp::arg("self"), bp::arg("log_cholesky_parameters"))))
```

Comment:
There is no runtime check of the vector input size. This can make the interpreter crash.